### ISSUE TITLE
Error when searching or returning hostname

### ISSUE THREAD

**Issue by k3nd0r** [swisscom/PowerGRR#26]
I am having an error when running commandlets that will search by hostname or return a hostname value such as: Find-GRRClient and Get-GRRClientIdFromComputerName

The exact commands I am attempting to run with these, including options, are:

`Find-GRRClient REDACTED -Credential $GRRCredential -OnlyComputerName`
and
`Get-GRRClientIdFromComputerName -ComputerName REDACTED`

The errors returned from these commands are the:
<pre>
The property 'node' cannot be found on this object. Verify that the property exists.
At \\REDACTED\WindowsPowerShell\Modules\PowerGRR\0.9.0\PowerGRR.psm1:427 char:17
+                 $ret.items.os_info.node
+                 ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict
</pre>
and
<pre>
The property 'node' cannot be found on this object. Verify that the property exists.
At \\REDACTED\WindowsPowerShell\Modules\PowerGRR\0.9.0\PowerGRR.psm1:348 char:25
+                         $info=[ordered]@{
+                         ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict
</pre>

I have tried looking through documentation to see what the error would be but have not been able to find a solution.

**Comment by k3nd0r** [comment]
I have been successful when running the command, so I know that I am authenticating properly and that my search will return results that I am expecting:

`Find-GRRClient REDACTED -Credential $GRRCredential`

**Comment by k3nd0r** [comment]
Additionaly,

When I run the command:

`Get-GRRClientInfo -ComputerName REDACTED`

I get the same error, followed by some information on the client, but the ComputerName field is empty. Fields that are returned with correct information are ClientId, InstallationDate, LastSeenAt, OSVersion, GRRClientVersion, UserNames, Labels

**Comment by Karneades** [comment]
@k3nd0r I'm very sorry about the delay. I fixed the issue a month ago in Github - but not in a release yet. See https://github.com/swisscom/PowerGRR/commit/92b34e4b39bcf0500b01f896c0e803aa9b1fd3fc. 

Could you shortly confirm that the issue is fixed with the newest version from Github?

**Comment by Karneades** [comment]
> Find-GRRClient REDACTED -Credential $GRRCredential

@k3nd0r If you use the variable named `$GRRCredential` for your credentials you don't need to provide the `-Credential` parameter afterwards for the subsequent commands.

**Comment by k3nd0r** [comment]
@Karneades 

No worries about the delay. I have cloned the repo and imported the module successfully into powershell. I have just started testing some stuff but it looks like the errors that I was receiving have been fixed due to your update. I not longer receive the: "The property 'node' cannot be found"... issue anymore.

Looking at your commit, seems like that .fqdn did the trick versus the .node which lines up with my error.

Thanks for the help, I really like the module and am looking forward to using it in the future!

I will close this and if I run into any related issues to this I will update you. Thanks again.
